modules/gltf/src/gltf-loader.js

```diff
@@ -16,23 +15,12 @@ const GLTFLoader = {
 
   options: {
     gltf: {
-      parserVersion: 1, // the new parser that will be the only option in V2 is not default in V1
```

Comment:
So gltf loader will be default to `parserVersion: 2`?

Reply:
Correct, the v1 loader is completely removed now, so no need to specify version.